core/administration.go

```diff
@@ -21,7 +21,7 @@ func (app *Application) getTODO() error {
 }
 
 func (app *Application) getGroupsUnprotected(clientID string, category *string, privacy *string, title *string, offset *int64, limit *int64, order *string) ([]model.Group, error) {
-	groups, err := app.storage.FindGroups(clientID, category, privacy, title, offset, limit, order)
+	groups, err := app.storage.FindGroups(clientID, nil, category, privacy, title, offset, limit, order)
```

Comment:
Shouldn't something similar be used for the admin/v2/groups endpoint? Currently it is still filtering by user.

Reply:
The "administrator" should access everything (through the admin client) - this is the reason why not honouring the privacy, hidden flag and the members list.